tests/mock_tables/state_db.json

```diff
@@ -708,5 +708,11 @@
         "non_fatal|Undefined": "0",
         "non_fatal|UnsupReq": "0",
         "non_fatal|UnxCmplt": "0"
+    },
+    "MUX_METRICS_TABLE|Ethernet0": {
+        "linkmgrd_switch_active_start": "2021-May-13 10:00:21.420898",
```

Comment:
is it possible the same date format as does linkmgrd, `yyyy-mmm-dd`?

Reply:
It's actually taken care in the xcvrd now, this version was running on old xcvrd changes, I will update the comment and testcases. 

`admin@STR43-0101-0101-01LT0:~$ redis-cli -n 6 hgetall "MUX_METRICS_TABLE|Ethernet0"
 1) "linkmgrd_switch_active_start"
 2) "2021-May-13 10:00:21.420898"
 3) "linkmgrd_switch_unknown_end"
 4) "2021-May-13 10:00:26.123319"
 5) "xcvrd_switch_standby_start"
 6) "2021-May-17 18:15:11.057528"
 7) "xcvrd_switch_standby_end"
 8) "2021-May-17 18:15:11.062379"
 9) "linkmgrd_switch_standby_end"
10) "2021-May-17 18:15:11.062887"`